src/assets/styles/var.sass

```diff
@@ -137,7 +137,7 @@ $app-color-action-list-item-inline-divider: #d0d7de7a
     border-radius: 1px
     position: relative
     float: left
-    border-color: $app-color-border-default
+    border-color: $app-color-border-default!important
     height: inherit
     background: $app-aliceBlue
```

Comment:
Just want to mention there is an error value in the variable, there are double background and it doesn't work.
Could you also fix this?